tests/integration/memcache_tests.py

```diff
@@ -41,7 +41,7 @@ def test_simple(self):
 
         server_span_observer = self.baseplate_observer.get_only_child()
         span_observer = server_span_observer.get_only_child()
-        self.assertEqual(span_observer.span.name, "memcache.get")
+        self.assertEqual(span_observer.span.name, "memcache")
```

Comment:
Hm, why remove the method name from the span name?

 When viewing a trace in zipkin it's really useful to be able to see the basic details in the span name since that's what's visible in the tree of spans and you have to click through to an individual span to see its tags. 

(Ah, I see the discussion of this in the ticket. I think it'd be better to bring r2 up to spec with how baseplate names its spans rather than reduce baseplate to r2's level)